portworx-lighthouse/step4.md

```diff
@@ -0,0 +1 @@
+>>Q9: Update the volume 'local_volume' and set the property shared to 'True'<<
```

Comment:
Note, the volume needs to be detached to do this operation. Also, "set the property to true" might be misleading as the command is `on/off` example. ` pxctl volume update --shared=on testvol` Maybe just say update the volume so it is a shared volume or something so they can do it with `true` in the pvc or via pxctl

Reply:
I have updated the wording to say 'on' instead of true.